README.rst

```diff
@@ -86,6 +86,15 @@ Thanks goes to all people that make scikit-matter possible:
    :alt: Code coverage
    :target: https://codecov.io/gh/scikit-learn-contrib/scikit-matter/
 
+.. |python| image:: https://img.shields.io/badge/documentation-latest-sucess
+   :alt: Python
+   :target: https://scikit-matter.readthedocs.io
+
+.. |python| image:: https://img.shields.io/badge/python-v3.8,_v3.9,_v3.10,_v3.11-blue.svg
+   :alt: Tested Python versions in CI
+   :target: (https://github.com/scikit-learn-contrib/scikit-matter/\
+                blob/main/.github/workflows/tests.yml)
```

Comment:
We had this idea also for equistore and rascaline. But, we concluded that a version banner does not add much additional information and is cluttering the README. Especially because on PYPI the versions will now be shown in the Meta section (see i.e. [scikit-learn](https://pypi.org/project/scikit-learn/)) based on your additions below.

Reply:
I would say it is not exactly the same information as in the Meta section on pypi. "Python (>= 3.8)" does not specify which version we test on in the CI. I am not sure if the link is very meaningful. Maybe we can combine it with the test badge that we test on these versions. I will look into this.